react/wbooks/src/app/index.js

```diff
@@ -11,11 +10,9 @@ import PublicRoute from './components/PublicRoute';
 function App() {
   return (
     <Router>
-      <div className={styles.container}>
-        <PublicRoute exact path="/" component={Login} redirect="/home" />
-        <PrivateRoute exact path="/home" component={Home} redirect="/" />
-        <PublicRoute path="/sign_up" component={SignUp} redirect="/home" />
-      </div>
+      <PublicRoute exact path="/" component={Login} redirect="/home" />
+      <PrivateRoute exact path="/home" component={Home} redirect="/" />
+      <PublicRoute path="/signUp" component={SignUp} redirect="/home" />
```

Comment:
specify the redirect only when neccesary. And keep '/home' and '/' as the default redirect on the `Route` component